apps/dsb-client-gateway-api/src/app/modules/message/service/message.service.ts

```diff
@@ -667,6 +730,47 @@ export class MessageService {
       }
     }
 
+    if (channel.messageForms) {
+      this.logger.debug(`attempting to store received messages`);
```

Comment:
I think this is unreachable, the same check is performed above (L596) and returns, so this code is never reached